api/suite_test.go

```diff
@@ -97,6 +97,8 @@ func (s *S) SetUpSuite(c *check.C) {
 	app.AuthScheme = nativeScheme
 	p := app.Platform{Name: "zend"}
 	s.conn.Platforms().Insert(p)
```

Comment:
That's fine, if you prefer, you can add the new platform only in the test that you're going to use it.